tests/Console/ModelsCommand/DoesNotGeneratePhpdocWithExternalEloquentBuilder/__snapshots__/Test__test__1.php

```diff
@@ -90,79 +90,79 @@
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTextNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTextNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTimeNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTimeNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTimestampNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTimestampNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTimestamptzNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTimestamptzNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTimetzNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTimetzNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTinyIntegerNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereTinyIntegerNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedBigIntegerNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedBigIntegerNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedDecimalNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedDecimalNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedIntegerNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedIntegerNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedMediumIntegerNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedMediumIntegerNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedSmallIntegerNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedSmallIntegerNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedTinyIntegerNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUnsignedTinyIntegerNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUpdatedAt($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUuidNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereUuidNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereYearNotNullable($value)
- * @method static \Illuminate\Database\Eloquent\Builder|Post whereYearNullable($value)
+ * @method static \Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\DoesNotGeneratePhpdocWithExternalEloquentBuilder\Builders\PostExternalQueryBuilder|Post whereBigIntegerNotNullable($value)
```

Comment:
@mfn I do not know if it is the right thing. But perhaps if they deactivate the external builder option, we should not use the external builder in the where magic methods? Let me know what you think.

Reply:
@KaloyanYosifov thank for mentioning it, in fact I already wrote this during my review but re-arranged it now for context:

> Is this test here obsolete?
> 
> Because it says (the namespace, I separated the words): "Does Not Generate Phpdoc With External Eloquent Builder"
> 
> But OTOH I'm unsure, it looks all the same to me with the test in `GeneratePhpdocWithExternalEloquentBuilder`, would appreciate any insights.

So I guess, yes, it shouldn't?

Reply:
@mfn Thank you for your time on this review 🙏 . I think we are going to need this test as we have a new option that defines if we should either use the "external eloquent builder methods" feature or not. And since that option "write_model_external_builder_methods" is true by default, we are disabling it in this [Test](https://github.com/barryvdh/laravel-ide-helper/blob/master/tests/Console/ModelsCommand/DoesNotGeneratePhpdocWithExternalEloquentBuilder/Test.php#L16) to check if it is working correctly (not using the external builder).